Lombiq.NodeJs.Extensions/config/lombiq.markdownlint.json

```diff
@@ -1,3 +1,4 @@
+// When updating, also update the same configuration file accompanying the "markdown-lint" action in Lombiq.GitHub.Actions.
```

Comment:
Are you sure this works? Last time I checked _.markdownlint.json_ has to be a standard JSON file (meaning it has to work when passed into `JSON.parse()` in Javascript), so comments will break the parser.